### PR TITLE
[patch] Logging Delete ActiveEnvironment API

### PR DESCRIPTION
**Pull request by sunny299**
**What's the change or impact of this PR?**:
- add fields activeDeletedBy and activeDeletedAt in team component
- call cinterny to store active namespace deletion information into database

**Related issue link**:

### CODE REVIEW

**Review by quraynai — APPROVED** [review]

File: api/v1beta1/config_types.go [hunk 1]
```diff
@@ -307,6 +307,8 @@ type ReporterShell struct {
 	PullRequestTrigger *CommandAndArgs `json:"pullRequestTrigger,omitempty"`
 	// +optional
 	PullRequestQueue *CommandAndArgs `json:"pullRequestQueue,omitempty"`
+	// +optional
+	ActiveNamespaceDeleted *CommandAndArgs `json:"activeNamespaceDeleted,omitempty"`
```
Comment (Pohfy123) [inline api/v1beta1/config_types.go:311]: Can we rename to `ActiveEnvironmentDeleted` ?

File: internal/reporter.go [hunk 2]
```diff
@@ -201,6 +202,26 @@ func NewPullRequestTriggerResultReporter(status s2hv1beta1.PullRequestTriggerSta
 	return c
 }
 
+// DeletedActiveNamespaceReporter manages active namespace deletion report
+type DeletedActiveNamespaceReporter struct {
```
Comment (Pohfy123) [inline internal/reporter.go:206]: Can we rename to `DeletedActiveEnironmentReporter`

File: internal/reporter.go [hunk 3]
```diff
@@ -12,11 +12,12 @@ import (
...
-	ActivePromotionType    EventType = "ActivePromotion"
-	ImageMissingType       EventType = "ImageMissing"
-	PullRequestTriggerType EventType = "PullRequestTrigger"
-	PullRequestQueueType   EventType = "PullRequestQueue"
+	ComponentUpgradeType       EventType = "ComponentUpgrade"
+	ActivePromotionType        EventType = "ActivePromotion"
+	ImageMissingType           EventType = "ImageMissing"
+	PullRequestTriggerType     EventType = "PullRequestTrigger"
+	PullRequestQueueType       EventType = "PullRequestQueue"
+	ActiveNamespaceDeletedType EventType = "ActiveNamespaceDeleted"
```
Comment (Pohfy123) [inline internal/reporter.go:20]: Can we rename to `ActiveEnvironmentDeletedType`

File: api/v1beta1/team_types.go [hunk 4]
```diff
@@ -270,6 +274,7 @@ const (
 	TeamUsedUpdated                       TeamConditionType = "TeamUsedUpdated"
 	TeamRequiredFieldsValidated           TeamConditionType = "TeamRequiredFieldsValidated"
 	TeamActiveEnvironmentDelete           TeamConditionType = "TeamActiveEnvironmentDelete"
+	TeamSentDeletedActiveNamespace        TeamConditionType = "TeamSentDeletedActiveNamespace"
```
Comment (Pohfy123) [inline api/v1beta1/team_types.go:277]: Should we rename to `TeamActiveEnvironmentDeletedReportSent`?

File: api/v1beta1/team_types.go [hunk 5]
```diff
@@ -270,6 +274,7 @@ const (
 	TeamUsedUpdated                       TeamConditionType = "TeamUsedUpdated"
 	TeamRequiredFieldsValidated           TeamConditionType = "TeamRequiredFieldsValidated"
 	TeamActiveEnvironmentDelete           TeamConditionType = "TeamActiveEnvironmentDelete"
```
Comment (Pohfy123) [inline api/v1beta1/team_types.go:276]: Can we rename this also `TeamActiveEnvironmentDeleted`?
Would it affect the current samsahai?

Comment (sunny299) [reply api/v1beta1/team_types.go:276]: should not affect ka

**Review by Pohfy123 — APPROVED** [review]

**Pull request merged**